index.html

```diff
@@ -290,7 +290,21 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
-
+    <footer>
+      <section>
+        <form class="footerForm">
```

Comment:
- [ ] OPTIONAL
- Let us collect users' data using [formspree](https://formspree.io/html). I am sorry the previous reviewer did not notice this, and therefore, I will mark it as optional. Collecting users' data helps with interactions between you and the users of your product who could be your future clients or employees as is in this portfolio site.